cmd/goa4web/user_make_admin.go

```diff
@@ -26,11 +26,21 @@ func parseUserMakeAdminCmd(parent *userCmd, args []string) (*userMakeAdminCmd, e
 		return nil, err
 	}
 	c.fs = fs
+	args = fs.Args()
+	if c.Username == "" && len(args) > 0 {
```

Comment:
This is supposed to be making it more consistent with the other tools with how it reports it's usage and hints etc.

This is good but we need more usage help